perma-payments/fabfile.py

```diff
@@ -0,0 +1,66 @@
+# set up Django
+import os
+import django
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'config.settings')
+try:
+    django.setup()
+except Exception as e:
```

Comment:
wondering why print exception as opposed to raise Exception

Reply:
This is actually how Perma and Capstone are configured too. (https://github.com/harvard-lil/perma/blob/develop/perma_web/fabfile/__init__.py#L9, https://github.com/harvard-lil/capstone/blob/develop/capstone/fabfile.py#L10). Looks like its a brain child of Jack's

Here's how I read it: we're printing, rather than attempting to log, because we want the message to appear right in the developer's console when they are running the fab task. stout/sterr sounds to me like the right move here, rather than a log call: I'm not sure where fab logs go, by default. If I had to guess, straight to /dev/null lol. Since we only run fab tasks locally, from our own machines, it probably doesn't make sense to configure any default logging, though I could imagine that changing if we had some central log aggregation going on (e.g., any time any dev runs a fab tasks that targets prod, the logs get sent to Splunk or what have you....)

And I think this try/except clause exists so that we could, theoretically, run fab tasks that don't depend on Django, and run them even if the Django app is struggling for whatever reason. For example, the fab task to turn on maintenance mode, which is accomplished via bash and Nginix, shouldn't be dependent on the Django app starting up successfully.